app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  def index
+    tickets = Ticket.all
+    render json: tickets
+  end
+
+  def show
+    ticket = Ticket.find(params[:id])
+    render json: ticket
+  end
+
+  def create
+    ticket = Ticket.new(ticket_params)
+    if Ticket.where.not(Ticket.screening.id = ticket.screening.id, Ticket.seat.name = ticket.seat.name)
```

Comment:
Does this work? Btw I think you're looking for a constraint or some model validations?

Reply:
I wanted to check if there is a record in Ticket db that has the same screening id and seat name as the one Im trying to create in line 15 and if not let it pass and save

Reply:
There is a [exists?](https://apidock.com/rails/ActiveRecord/FinderMethods/exists%3F), you can check it using:
```
Ticket.exists?(id: ..., ...)
```

If you need to build a condition checking data from more than one table you need to use `joins` or `includes`, we discussed it during [My last presentation]( https://docs.google.com/presentation/d/1lSFSp8KzoV8vFS605IThENS-uAsa1PK1GgysHRbq_gw/edit#slide=id.gdafe97035b_0_0)

The syntax doesn't look valid. How do you test the code? Do you use postman/curl/anything like that?